src/components/transaction/TransactionBlock.scss

```diff
@@ -0,0 +1,38 @@
+#transaction-block {
+  background: rgba(255, 255, 255, 0.07);
+  border-radius: 2.4px;
+  display: flex;
+  flex: 1 1;
```

Comment:
Yumi added the `horiz`, `verti`, and `weight-?` to the project
check if it's possible to reuse them